operator/cmd/mesh/testdata/manifest-migrate/output/overlay_values.golden.yaml

```diff
@@ -7,6 +7,9 @@ spec:
     ingressGateways:
     - enabled: true
       k8s:
+        hpaSpec:
+          maxReplicas: 20
+          minReplicas: 20
```

Comment:
where's the input change corresponding to this output change?

Reply:
It is because I fix the logic in the reverse translation code, previously it would do translate only when autoscaleEnabled explicitly defined as true, but now I updated the logic to "skip translate" when autoscaleEnabled explicitly defined as false